Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -534,3 +575,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         return df
     }()
 }
+
+private extension UICollectionView {
+    var isLastCellVisible: Bool {
+        if numberOfItems(inSection: 0) == 0 { return true }
+        let lastIndexPath = IndexPath(item: 0, section: 0)
+
+        guard let attributes = collectionViewLayout.layoutAttributesForItem(at: lastIndexPath) else { return false }
+        return bounds.contains(attributes.frame)
```

Comment:
```suggestion
        let lastIndexPath = IndexPath(item: 0, section: 0)
        return collectionView.indexPathsForVisibleItems.contains(lastIndexPath)
```

This should be enough, I've been using this 👍

Reply:
I had this option listed before but I already removed it, sorry. 
Your suggested change will be true immediately when the cell will appear on the screen, even when only partially visible, but the code I'm using will be true only when the full cell is visible, which is what I opted for here.

Reply:
Ah I see... In my case, I actually just want the cell partially visible. Maybe rename this to `isLastCellFullyVisible`? 🤔  I don't know 😄

Reply:
I had this exact name with an extra getter 
```
var isLastCellVisible: Bool { isLastCellFullyVisible }
```
And removed it to siplify the code 😅
I will think about it until tomorrow 👍🏻